src/ripple/app/consensus/RCLConsensus.cpp

```diff
@@ -746,7 +747,7 @@ RCLConsensus::Adaptor::buildLCL(
         closeTimeCorrect = ((replay->closeFlags_ & sLCF_NoConsensusTime) == 0);
     }
 
-    JLOG(j_.debug()) << "Report: TxSt = " << set.id() << ", close "
+    JLOG(j_.debug()) << "Report: TxSt = " << txns.id() << ", close "
                      << closeTime.time_since_epoch().count()
                      << (closeTimeCorrect ? "" : "X");
```

Comment:
While you're here... how about:

    (closeTimeCorrect ? "" : " (incorrect)")

Just slapping an `X` after a long string of number makes no sense.